W-13734453-glossary-update-cv (#212)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/glossary.adoc

== A

''''''

=== Action Step

The smallest components in a Workflow. Use action steps to build logical blocks and transactions in RPA Builder. 

=== Activity

A single unit of work in the automation project. An activity can be a bot task and its linked workflow, or a user task.

=== Activity Library 

> Enables you to reuse certain activities that repeat during the automation of business processes, such as a login, to avoid creating the same workflow multiple times.

Reviewer: Description not parallel with the previous glossary entries, which are noun phrases. Probably should be similar. Also see the [Anypoint Platform glossary](https://docs.mulesoft.com/general/glossary).

=== Activity Parameter 

> Enables you to map variables that you can use not only in the Workflow, but also in the activities of the business process and gateways.

Reviewer: See line 21.

> Enables you to map variables that you can use not only in the Workflow, but also in the activities of the business process and gateways.

Reviewer: ```suggestion
Enables you to map variables that you can use both in the Workflow and in the activities of the business process and gateways.
```

Reviewer: (avoid negative language)

=== Analysis Package 

> An error report that contains full information on the run in the same format as when you execute the Workflow locally.

Reviewer 2: Not a 'report'. It's a *.zip file containing logfile of the workflow run and error screenshots. When opened / imported in RPA Builder it will display the info in the same format as when executed locally in Builder.

> An error report that contains full information on the run in the same format as when you execute the Workflow locally.

Author: ```suggestion
A ZIP file containing workflow run logs and error screenshots. Importing the package in RPA Builder shows information in the same format as when the workflow executes locally.
```

''''''

> === IDP 

Reviewer: ```suggestion
=== IDP (Intelligent Document Processing)
```

> Intelligent Document Processing is a technology solution that implements AI to read documents, extract data, and organize information from different document sources.

Reviewer: ```suggestion
A technology solution that implements AI to read documents, extract data, and organize information from different document sources.
```

Reviewer: Suggestions to make descriptions parallel.

=== Invokable Run Configuration 

> An automated process that you can upload to exchange and then invoke from other services as Composer. 

Reviewer 2: The configuration is created in RPA Manager and with saving it, it will automatically uploading an invokable API in the exchange of the anypoint org.
So "automated process" is not the right term here. For more info see https://docs.mulesoft.com/rpa-home/invoke-rpa-services and https://docs.mulesoft.com/rpa-manager/processautomation-deploy#publish-automation-exchange

> An automated process that you can upload to exchange and then invoke from other services as Composer. 

Author: ```suggestion
A run configuration that RPA Manager uploads to Anypoint Exchange as an invokable API to be called by other services such as Composer.
```

== P

> === Properties 

Reviewer: Can you make singular?

== R

=== Regular Expression 

A sequence of characters and symbols that specify a match pattern to be searched in text.  

=== Run Configuration 

> A schedule that specifies the times and intervals for a process to run.

Author: ```suggestion
A set of execution parameters and run conditions for an automated process that includes a name, run schedule, activity parameters' starting values, and assigned user tasks and bots.
```

> 

Reviewer 2: A Run configuration contains more than just the schedule, it also have a name, set the activity parameter starting values, assign user tasks and assign bots. See https://docs.mulesoft.com/rpa-manager/processautomation-deploy#production-configuration

''''''
=== RPA

MuleSoft Robotic Process Automation (RPA) enables you to automate business processes and tasks by using bots, which helps save time and prevent introducing human errors.

> === RPA Bots

Reviewer: ```suggestion
=== RPA Bot
```

> RPA Bots are software robots that run in an on-premises environment and execute the assigned automations. You can manage bots from RPA Manager.

Reviewer: ```suggestion
A software robot that runs in an on-premises environment and executes the assigned automation. You can manage bots from RPA Manager.
```

Reviewer: Changed to singular to be parallel.

=== RPA Builder

> RPA Builder is an on-premises application that enables you to build the automations by specifying the steps of the process.

Reviewer: ```suggestion
An on-premises application that enables you to build automations by specifying the steps of the process.
```

=== RPA Manager

RPA Manager is the cloud-based control plane for the automations that is integrated into Anypoint Platform.

=== RPA Recorder

> RPA Recorder is an on-premises tool that assists during the process design phase by recording all user actions, creating a click path documentation and generating additional data to automatically create action steps in the build phase.

Reviewer: ```suggestion
An on-premises tool that assists during the process design phase by recording all user actions, creating click path documentation, and generating additional data to automatically create action steps in the build phase.
```
